clients/web/src/views/body/UsersView.js

```diff
@@ -63,6 +74,7 @@ var UsersView = View.extend({
         }));
 
         this.paginateWidget.setElement(this.$('.g-user-pagination')).render();
+        this.sortCollectionWidget.setElement(this.$('.g-user-sort')).render();
```

Comment:
We should add a little right-margin on this so it isn't right up against the search widget.